client/src/style/scss/custom_theme_variables.scss

```
@@ -2,6 +2,8 @@
 
 html,
 .reset-theme-variables {
+    --masthead-height: #{$masthead-height};
+    --masthead-logo-height: #{$masthead-logo-height};
```

```suggestion
    --masthead-logo-height: calc(var(--masthead-height) * 0.6);
```

calculating the logo height here, instead of in the scss variables would make setting it optional for most themes.